Tests/PointFreeTests/__Snapshots__/EpisodeTests/testEpisodeCredit_PrivateEpisode_NonSubscriber_HasCredits.1.Conn.txt

```diff
@@ -1916,7 +1916,8 @@ want to use it to view this episode for free right now?
         <div class="sticky-d top-0">
           <div class="w-100p relative" style="padding-bottom:56.25%">
             <video class="h-100p w-100p absolute bg-gray650" controls playsinline autoplay poster="">
-              <source src="">
+              <source src="https://s3.amazonaws.com/pointfreeco/trailer.m3u8">
```

Comment:
This changes the screen shots slightly...but not a problem?

Reply:
it may also cause some non-deterministic stuff with the screenshots. I remember running into that before, but we can try it out.